man/man8/zpoolconcepts.8

```diff
@@ -93,6 +93,56 @@ compromised.
 The minimum number of devices in a raidz group is one more than the number of
 parity disks.
 The recommended number is between 3 and 9 to help increase performance.
+.It Sy draid , draid1 , draid2 , draid3
+A distributed version of raidz with integrated hot spares that allows for
+faster resilvering.
+A draid vdev is constructed from multiple internal raidz groups, each with D
+data devices and P parity devices, which are distributed over the children.
+This allows all of the children to participate when resilvering.
+As a general rule the more devices in a draid the less time required to
+resilver to a distributed spare.
+Furthermore, a narrower stripe width (D+P) will improve resilver performance
+at the expense of total usable capacity.
+.Pp
+Like raidz a draid can have single-, double-, or triple-parity.  The
+.Sy draid1 ,
+.Sy draid2 ,
+and
+.Sy draid3
+types can be used to specified the required parity level.
+The
+.Sy draid
+vdev type is an alias for
+.Sy draid1 .
+.Pp
+A draid with N disks of size X, D data disks, P parity level, and H distributed
+hot spares can hold approximately (N-H)*(D/(D+P))*X bytes and can withstand P
+device(s) failing before data integrity is compromised.
+The minimum number of devices in a draid is P+2.
+.It Sy draid[<parity>][:<data>d][:<children>c][:<spares>s][:<iterations>i]
+When a non-default draid configuration is required it can be specified by
+appending one or more of the following optional arguments to the
+.Sy draid
+keyword.
+.Pp
+.Em children
+- The expected number of children.
+If the provided number of children is different an error is returned.
```

Comment:
So this is essentially an optional "checksum" on the number of provided disks?  Maybe we could explain the use case here, so that folks don't think that they need to specify it.  I'm imagining that the use case is where you've carefully thought about the layout and you want to make sure there isn't a mistake when listing the devices?

Reply:
That's right.  It's helpful as a cross check when you're working with many devices in a pool.  I'll provide a use case.